python/ray/tune/stopper.py

```diff
@@ -61,3 +64,61 @@ def is_valid_function(cls, fn):
                 "Stop object must be ray.tune.Stopper subclass to be detected "
                 "correctly.")
         return is_function
+
+
+class EarlyStopping(Stopper):
+    def __init__(self, metric, std=0.001, top=10, mode="min"):
+        """Create the EarlyStopping object.
+
+        Parameters
+        ---------------------
+        metric: str,
+            The metric to be monitored.
+        std: float = 0.001,
+            The minimal standard deviation after which
+            the tuning process has to stop.
+        top: int = 10,
+            The number of best model to consider.
+        mode: str = "min",
+            The mode to select the top results.
+            Can either be "min" or "max".
+
+        Raises
+        ----------------------
+        ValueError,
+            If the mode parameter is not "min" nor "max".
+        ValueError,
+            If the top parameter is not an integer
+            greater than 1.
+        ValueError,
+            If the standard deviation parameter is not
+            a strictly positive float.
```

Comment:
Can you convert this to google-style docstring?

Reply:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I do not know how to specify the raises in that format, do you already know a comprehensive example to start from? It would help a great deal.

Reply:
@richardliaw is it the current version correct?